cmd/main.go

```diff
@@ -23,106 +20,27 @@ var (
 	debug          = pflag.Bool("debug", false, "sets log to debug level")
 	versionInfo    = pflag.Bool("version", false, "prints the version information")
 	contextTimeout = pflag.Int("context-timeout", 110, "context timeout in seconds for provider calls")
+	endpoint       = pflag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
```

Comment:
nit: should we make the default name more descriptive e.g. `azure.sock` and description `CSI grpc endpoint`